### PR TITLE
Update vcperf-commands.md

### PR DESCRIPTION
**Pull request by NelsonDaniel**
Update commands documentation according to https://github.com/microsoft/vcperf#command-line-reference

### CODE REVIEW

**Comment by Jak-MS** [comment]
@kevcadieux 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/build-insights/reference/vcperf-commands.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ This article lists and describes the commands available in *`vcperf.exe`*, and h
 ## Commands to start and stop traces
 
 > [!IMPORTANT]
-> The following commands all require administrative privileges.
+> Unless /noadmin is provided, the following commands require administrative privileges.
```
Comment (kevcadieux) [inline docs/build-insights/reference/vcperf-commands.md:21]: Consider adding this note "Unless /noadmin is provided" in the vcperf documentation on GitHub as well.

**Review by kevcadieux — APPROVED** [review]

**Comment by kevcadieux** [comment]
@Jak-MS I reviewed the PR. I will let @NelsonDaniel sign off when he is ready.

**Comment by Court72** [comment]
Hi @kevcadieux 

Thanks for reviewing the PR! 

FYI - in the public repos, PRMerger will accept sign-off *from the listed author only* (hence the need for you to add a `#sign-off` comment to signal that the PR is ready for the review team to merge).

Thanks.

**Comment by corob-msft** [comment]
@NelsonDaniel 

Which version of the product do the /noadmin, /templates, and /levelN options appear in?

Stashing this update for Acrolinx and i18n here:

```
> [!IMPORTANT]
> Unless you specify `/noadmin`, the following commands require administrative privileges.

| Option           | Arguments and description |
|------------------|---------------------------|
| `/start`         | [`/noadmin`] [`/nocpusampling`] [`/level1` \| `/level2` \| `/level3`] `<sessionName>` |
|                  | Starts a trace under the given session name. <br/><br/> The `/noadmin` option runs *vcperf.exe* without admin privileges, and it ignores the `/nocpusampling` option. When you run vcperf without admin privileges, there can be more than one active session on a given machine. <br/><br/> The `/nocpusampling` option specifies *vcperf.exe* doesn't collect CPU samples. It prevents the use of the CPU Usage (Sampled) view in Windows Performance Analyzer, but makes the collected traces smaller. <br/><br/> The `/level1`, `/level2`, or `/level3` options specify which MSVC events to collect, in increasing level of information. Level 3 includes all events. Level 2 includes all events except template instantiation events. Level 1 includes all events except template instantiation, function, and file events. If unspecified, `/level2` is selected by default. <br/><br/> Once *vcperf.exe* starts tracing, it returns immediately. The trace collects events system-wide for all processes running on the machine. That means that you don't need to build your project in the same command prompt window as the one you use to run *vcperf.exe*. For example, you can build your project in Visual Studio. |
| `/stop`          | (1) [`/templates`] `<sessionName> <outputFile.etl>` <br/> (2) [`/templates`] `<sessionName> /timetrace <outputFile.json>` |
|                  | Stops the trace identified by the given session name. Runs a post-processing step on the trace to generate a file specified by the `<outputFile>` parameter. <br/><br/> The `/templates` option includes template instantiation events in the file. <br/><br/> (1) Generates a file viewable in Windows Performance Analyzer (WPA). The output file requires a `.etl` extension.<br/>(2) Generates a file viewable in the Microsoft Edge trace viewer (`edge://tracing`). The output file requires a `.json` extension. |
| `/stopnoanalyze` | `<sessionName> <rawOutputFile.etl>` |
|                  | Stops the trace identified by the given session name and writes the raw, unprocessed data in the specified output file. The resulting file isn't meant for viewing in WPA. <br/><br/> The post-processing step involved in the `/stop` command can sometimes be lengthy. You can use the `/stopnoanalyze` command to delay this post-processing step. Use the `/analyze` command when you're ready to produce a file viewable in Windows Performance Analyzer or the Microsoft Edge trace viewer. |

## Miscellaneous commands

| Option     | Arguments and description |
|------------|---------------------------|
| `/analyze` | (1) [`/templates`] `<rawInputFile.etl> <outputFile.etl>` <br/>(2) [`/templates`] `<rawInputFile.etl> /timetrace <outputFile.json>` |
|            | Accepts a raw trace file produced by the `/stopnoanalyze` command. Runs a post-processing step on this trace to generate the file specified by the `<outputFile>` parameter. <br/><br/> The `/templates` option includes template instantiation events in the file. <br/><br/> (1) Generates a file viewable in Windows Performance Analyzer (WPA). The output file requires a `.etl` extension. <br/><br/> (2) Generates a file viewable in the Microsoft Edge trace viewer (`edge://tracing`). The output file requires a `.json` extension. |

```

**Comment by NelsonDaniel** [comment]
> @NelsonDaniel
> 
> Which version of the product do the /noadmin, /templates, and /levelN options appear in?
> 
> Stashing this update for Acrolinx and i18n here:
> 
> ```
> > [!IMPORTANT]
> > Unless you specify `/noadmin`, the following commands require administrative privileges.
> 
> | Option           | Arguments and description |
> |------------------|---------------------------|
> | `/start`         | [`/noadmin`] [`/nocpusampling`] [`/level1` \| `/level2` \| `/level3`] `<sessionName>` |
> |                  | Starts a trace under the given session name. <br/><br/> The `/noadmin` option runs *vcperf.exe* without admin privileges, and it ignores the `/nocpusampling` option. When you run vcperf without admin privileges, there can be more than one active session on a given machine. <br/><br/> The `/nocpusampling` option specifies *vcperf.exe* doesn't collect CPU samples. It prevents the use of the CPU Usage (Sampled) view in Windows Performance Analyzer, but makes the collected traces smaller. <br/><br/> The `/level1`, `/level2`, or `/level3` options specify which MSVC events to collect, in increasing level of information. Level 3 includes all events. Level 2 includes all events except template instantiation events. Level 1 includes all events except template instantiation, function, and file events. If unspecified, `/level2` is selected by default. <br/><br/> Once *vcperf.exe* starts tracing, it returns immediately. The trace collects events system-wide for all processes running on the machine. That means that you don't need to build your project in the same command prompt window as the one you use to run *vcperf.exe*. For example, you can build your project in Visual Studio. |
> | `/stop`          | (1) [`/templates`] `<sessionName> <outputFile.etl>` <br/> (2) [`/templates`] `<sessionName> /timetrace <outputFile.json>` |
> |                  | Stops the trace identified by the given session name. Runs a post-processing step on the trace to generate a file specified by the `<outputFile>` parameter. <br/><br/> The `/templates` option includes template instantiation events in the file. <br/><br/> (1) Generates a file viewable in Windows Performance Analyzer (WPA). The output file requires a `.etl` extension.<br/>(2) Generates a file viewable in the Microsoft Edge trace viewer (`edge://tracing`). The output file requires a `.json` extension. |
> | `/stopnoanalyze` | `<sessionName> <rawOutputFile.etl>` |
> |                  | Stops the trace identified by the given session name and writes the raw, unprocessed data in the specified output file. The resulting file isn't meant for viewing in WPA. <br/><br/> The post-processing step involved in the `/stop` command can sometimes be lengthy. You can use the `/stopnoanalyze` command to delay this post-processing step. Use the `/analyze` command when you're ready to produce a file viewable in Windows Performance Analyzer or the Microsoft Edge trace viewer. |
> 
> ## Miscellaneous commands
> 
> | Option     | Arguments and description |
> |------------|---------------------------|
> | `/analyze` | (1) [`/templates`] `<rawInputFile.etl> <outputFile.etl>` <br/>(2) [`/templates`] `<rawInputFile.etl> /timetrace <outputFile.json>` |
> |            | Accepts a raw trace file produced by the `/stopnoanalyze` command. Runs a post-processing step on this trace to generate the file specified by the `<outputFile>` parameter. <br/><br/> The `/templates` option includes template instantiation events in the file. <br/><br/> (1) Generates a file viewable in Windows Performance Analyzer (WPA). The output file requires a `.etl` extension. <br/><br/> (2) Generates a file viewable in the Microsoft Edge trace viewer (`edge://tracing`). The output file requires a `.json` extension. |
> ```

Here is the release version: https://github.com/microsoft/vcperf/releases/tag/2.1.22062801

**Review by corob-msft — APPROVED** [review]

**Comment by corob-msft** [comment]
@NelsonDaniel Thanks for the update. I assume since @kevcadieux made only one comment that it otherwise passed his review, so I merged it.

**Pull request merged**